Updated Starlette/FastAPI setup documentation (#5472)
getsentry/sentry-docs: src/platforms/python/guides/fastapi/index.mdx

The FastAPI integration adds support for the [FastAPI Framework](https://fastapi.tiangolo.com/).

> Sentry SDK will auto enabled support for FastAPI if you have the `fastapi` Python package installed in your project. There is no configuration options you need to add when initializing the Sentry SDK, everything works out of the box.

Reviewer: ```suggestion
The Sentry SDK automatically enables support for FastAPI if you have the `fastapi` Python package installed in your project. There are no configuration options you need to add when initializing the Sentry SDK as everything works out of the box.
```

## Options

If you want to change the default behavior of the FastAPI integration, you need to instantiate the integration by hand and pass it to Sentry's `init` function.

> Because FastAPI is based on the Starlette framework both integrations `StarletteIntegration`, and `FastApiIntegration` must be instantiated.

Reviewer: ```suggestion
Because FastAPI is based on the Starlette framework, both integrations `StarletteIntegration` and `FastApiIntegration`, must be instantiated.
```

> You can pass the following keyword arguments to `StarletteIntegration()` respectively `FastApiIntegration()`:

Reviewer: I'm assuming this was copied from somewhere where there were multiple options in the list below so the order of them mattered, in which case, the word "respectively" would make sense. But here it doesn't work because there's only one option, so there's no order to respect. Also, we shouldn't have a bullet list with only one bullet item.

```suggestion
You can pass the keyword argument `transaction_style` to `StarletteIntegration()` and `FastApiIntegration()`.
```

> - `transaction_style`:

Reviewer: ```suggestion
```
